10up/restricted-site-access
# Fix: undefined nonce in settings js

### Description of the Change
This PR fixes a bug introduced in #123 that causes the `rsaAdmin` variable undefined, so users can't add new IPs because the nonce is undefined. Thank @paulschreiber for pointing it out.

This PR use separated localized variables for admin and settings and only load settings.js file on the Reading screen.



### Alternate Designs



We can still use one variable `rsaAdmin` for both js file, but I prefer less coupling solution.

### Verification Process

1. Go to Settings > Reading.
2. Whitelist an IP.
3. See no error, the IP is added to the list.



### Checklist:
